pkg/creds/creds.go

```diff
@@ -9,6 +9,7 @@ type LoginDetails struct {
 	Username     string
 	Password     string
 	MFAToken     string
+	DuoMFAOption string
 	URL          string
```

Comment:
I like to think this could be just MFAOption, leaving it open to be implemented on other providers?